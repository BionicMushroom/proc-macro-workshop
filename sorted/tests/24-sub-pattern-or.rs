#[sorted::check]
fn main() {
    let x = 5;

    #[sorted]
    match x {
        5 => println!("5"),
        7 | 8 | 6 => println!("6, 7 or 8"),
        _ => println!("other"),
    }
}
