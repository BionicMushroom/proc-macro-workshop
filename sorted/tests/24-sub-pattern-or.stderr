error: 6 should sort before 7
 --> tests/24-sub-pattern-or.rs:8:17
  |
8 |         7 | 8 | 6 => println!("6, 7 or 8"),
  |                 ^
